_layouts/indicator.html

```diff
@@ -171,5 +173,29 @@ <h4>{{ t.indicator.sub_categories }}</h4>
   </div>
 </div>
 
-{% include footer.html %}
+<style type="text/css">
+  #chartControls[show="1"]{display: block;}
```

Comment:
Do you think it might be possible to get a little fancy here, and to "grey out" the toolbar instead of hiding it altogether? I think that `pointer-events:none` would effectively disable it, and maybe an "overlay" div or something to grey out the contents? On desktop it just looks a little unexpected to have that chunk of the screen just be white.